README.md

```diff
@@ -222,69 +215,24 @@ func obtainJWTTokenFromExternalSources() -> DelegatedAuthClosure {
-- [Release Candidate (RC)](#release-candidate-rc)
-- [Current Release](#current-release)
-- [Long Term Support](#long-term-support)
+### Supported Version
 
-### Release Candidate (RC)
-The initial releases of the SDK starting Oct 19th will be Release Candidate (RC). This means (1) the core functionality is present and tested, (2) additional functionality from this point on will be considered improvements or enhancements based on customer feedback. RC releases are usually more frequent (every few weeks), followed shortly by a current release. If you plan to use an RC release, we recommend:
-- that you don't use it for production workloads (If that is unavoidable, we recommend upgrading to the Current Release version once it's ready.)
-- that you create a plan to keep your application on the latest RC release version at all times (since older RC releases are considered "out of support" as soon as a new RC release is released)
+Only the current MAJOR version of SDK is supported. New features, functionality, bug fixes, and security updates will only be added to the current MAJOR version.
 
-Also, RC releases may carry breaking changes from the previous release and we advise developers to test their application adequately with the new RC release before adopting it.
+A current release is on the leading edge of our SDK development, and is intended for customers who are in active development and want the latest and greatest features.  Instead of stating a release date for a new feature, we set a minor or path release cadence of 2-3 months. At the same time, there is no schedule for major or breaking release. Instead, we will communicate one quarter in advance the upcoming breaking change to allow customers to plan for the upgrade. We always recommend that all users run the latest available minor release for whatever major version is in use. We highly recommend upgrading to the latest SDK major release at the earliest convenient time and before the EOL date.
 
-The idea behind releasing RC releases is to rapidly iterate on the SDK (bug fixes, feature tweaks, etc.) to get it to a production-ready state, and typically we don't expect to have the SDK in the RC phase for more than a month.
+### Version schedule
 
-> #### Support for RC releases
-> A RC release
-> - is Considered [Active](#active) when released
-> - transitions to [End-of-life](#end-of-life) when the next release becomes Active
+| Version | Current Patch/Minor | Supported Environments                                  | State     | First Release | EOL/Terminated |
+|---------|---------------------|---------------------------------------------------------|-----------|---------------|----------------|
+| 5       | 5.1.0               | iOS 11.0+ / Mac OS X 10.13+ / tvOS 11.0+ / watchOS 4.0+ | Supported | 28 Oct 2021   | TBD            |
+| 4       | 4.4.0               |                                                         | EOL       | 13 Feb 2020   | 30 July 2022   |
```

Comment:
Is EOL/Terminated "30 July 2022" for version 4.4.0 is OK? Because now we have here ambiguity with it's state which is EOL.

Reply:
I am thinking about stating official EOL date in 3 months as we previously haven't communicated clearly EOL and version support. So saying that 4..4.0 is EOL on July 30th 2022 and until then it is not under active development/support.

Do you think we can communicate date when this policy is effective differently?